app/javascript/tests/subject/MoveToHousehold.test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { shallow } from 'enzyme';
+import { Button, Modal, Form } from 'react-bootstrap';
+import ReactTooltip from 'react-tooltip';
+import MoveToHousehold from '../../components/subject/MoveToHousehold.js'
+import { mockPatient1 } from '../mocks/mockPatients'
+
+const authyToken = "Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==";
+
+function getWrapper(patient) {
+    return shallow(<MoveToHousehold patient={patient} authenticity_token={authyToken} />);
+}
+
+function getPatientName(patient) {
+  return `${patient.first_name} ${patient.middle_name} ${patient.last_name}`;
+}
+
+describe('MoveToHousehold', () => {
+    it('Properly renders Move To Household button', () => {
+        const wrapper = getWrapper(mockPatient1);
+        expect(wrapper.find(Button).length).toEqual(1);
+        expect(wrapper.find(Button).text().includes('Move To Household')).toBeTruthy();
+        expect(wrapper.find('i').hasClass('fa-house-user')).toBeTruthy();
+        expect(wrapper.find(Button).prop('disabled')).toBeFalsy();
+        expect(wrapper.find(ReactTooltip).exists()).toBeFalsy();
+    });
+
+    it('Clicking the Move to Household button opens modal and modal renders correctly', () => {
```

Comment:
I'd break this up into an open modal test and a render modal test - that's how we've been doing it

Reply:
done here: https://github.com/SaraAlert/SaraAlert/pull/531/commits/b1f04abd405f230fca597fc31ec6d1652fefb39a